doc/_themes/sphinx13/static/sphinx13.css

```diff
@@ -411,11 +411,45 @@ div.viewcode-block:target {
 /* media queries */
 
 /* Reduce padding & margins for smaller screens */
-@media (max-width: 750px) {
+@media (max-width: 768px) {
     .sphinxsidebar {
         display: none;
     }
     div.body {
         border-left: none;
     }
 }
+
+
+/* Landing page */
+.sphinx-tagline * {
+    hyphens: none !important;
```

Comment:
I don't really like the usage of `!important` if it's not needed.

Reply:
It was needed; I don't feel it really here because it's a very specific CSS class that is only for our documentation, not part of a re-usable theme